bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -0,0 +1,1312 @@
+If you leave the option blank the binding will warn you in the logs and still use ffmpeg, so by adding this line it will remove the warning from your logs. 
+For cameras that have an API you can opt to not use the cameras stream and use ffmpeg instead by doing this as well should you need the streams for other reasons.
+
+FFmpeg may require you to lower the resolution and/or the FPS to lower the CPU load down enough to run, you may need to experiment.
+To change the settings used by this feature the binding exposes the config ``FFMPEG_MJPEG_ARGUMENTS`` which the default is currently ``-q:v 5 -r 2 -vf scale=640:-2 -update 1`` where 5 is the jpg quality/compression setting, and -r 2 is how many frames per second to try and create. In this case it will create 2 frames every second. -vf scale=640:-2 will lower the resolution down to make the video 640 pixels wide. You can remove this to use the same resolution as the camera is set to use, however it may become a trade off and you may get less frames per second if you raise the resolution.
+Always try to get the default settings working first before you begin to experiment and if your stream is above 1080p and 10 frames per second, consider lowering it if you have issues on an ARM based server like a raspberry PIx.
+ 
+
+
+## snapshots.mjpeg and autofps.mjpeg
+
+These similar features allow you to request a mjpeg stream created by the binding with low CPU usage from the cameras snapshots.
+Snapshots are usually high resolution and look great, however they are limited to a max of 1 frame per second (1 FPS).
+The reason this is more useful than snapshots on their own, is some UI's will flash white or black when a snapshot is refreshing, this does not happen with snapshots.mjpeg and is the same bandwidth and CPU load as just using snapshots!
+
+The autofps.mjpeg feature will display a snapshot that updates every 8 seconds to keep network traffic low, then when motion is detected it will automatically increase the frames to every second until the motion stops.
+This means lower traffic unless the picture is actually changing. 
+
+Request the stream to be sent to an item with this url. 
+NOTE: The IP is Openhabs not your cameras IP and the 54321 is what you have set as the SERVER_PORT.
+
+`http://192.168.xxx.xxx:54321/snapshots.mjpeg`
+
+Use the following to display it in your sitemap.
+
+```
+Video url="http://192.168.0.32:54321/autofps.mjpeg" encoding="mjpeg"
+
+Video url="http://192.168.0.32:54321/snapshots.mjpeg" encoding="mjpeg"
+```
+
+
+
+
+## HLS (HTTP Live Streaming)
+
+HLS is a way of splitting the live stream up into small h264 based files so it can be played in many browsers without using much CPU power and it can also contain audio. 
+Because the files need to be created and are not streamed live, this creates a lag/delay behind real time that can be lowered (more on that below). 
```

Comment:
You mentioned this already several times before. Maybe you can condense this a bit to keep this document as concise as possible.

Reply:
Reworked to make it shorter but certain key things need to be discussed as most users winge about the delays of HLS so it needs to be explained why it is harder to setup and why it has draw backs.